### PR TITLE
Few improvements to LS - see commits

### PR DESCRIPTION
**Pull request by vaijab**

### CODE REVIEW

**Review by Joseph-Irving — APPROVED** [review]
lgtm, though Marcin has a good point about running as non-root

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
-FROM quay.io/ukhomeofficedigital/centos-base:v0.2.0
+FROM fedora:25
```
Comment (marcinc) [inline Dockerfile:2]: Should we run it as non-root user?

**Review by marcinc — COMMENTED** [review]
One question. Otherwise LGTM.

File: Dockerfile (hunk 1 above)
Comment (Joseph-Irving) [reply Dockerfile:2]: that is a good point, I totally missed that

Comment (vaijab) [reply Dockerfile:2]: Would be great. I think it should mostly be possible, except for files in `/var/log/journal` and other hostPaths.

**Pull request merged**